js-pkg/react/src/main.tsx

```diff
@@ -185,6 +185,11 @@ export type YDocProviderProps = {
 
   /** Whether to hide the debugger link. Defaults to true. */
   showDebuggerLink?: boolean
+
+  /** Whether to enable offline support. Document state will be stored on the client side for offline use
+   * and faster startup times.
+   */
+  enableOfflineSupport?: boolean
```

Comment:
I was tempted by this, but I worry this implies that we are telling the component that it is offline and should act accordingly (i.e. don't even try to connect). Maybe just `offlineSupport`, since `enable` is implied by a boolean?

Reply:
Yeah I share that reservation. `offlineSupport` is fine, could we use the same name in the vanilla provider though?